lsp-clients.el

```diff
@@ -1029,5 +1029,17 @@ responsiveness at the cost of possible stability issues."
                                                             (parse-rf-language-server-include-path-regex
                                                              lsp-rf-language-server-include-paths))))))))))
 
+
+;; Nix
+(defgroup lsp-nix nil
+  "LSP support for Nix, using rnix-lsp."
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/nix-community/rnix-lsp"))
+
+(lsp-register-client
+ (make-lsp-client :new-connection (lsp-stdio-connection "rnix-lsp")
```

Comment:
You should probably defcustom this one so user can customize where the binary is.
Also using function instead of constant, so that when user change the binary path after `nix-ls` is loaded, it still works